best-practices/tech_lead.md

```diff
@@ -0,0 +1,313 @@
+
+A: As often as possible throughout the day. Absolute minimum at 2-3 times / day.  
+
+Q: _What does grooming the board mean?_
+- Is the ready column populated with prioritized, actionable work? (more below)
+- PRs needing review or re-review (more below)
+- Are open PRs associated with issues, if desired?
+- Are there issues in the new column that need to be triaged?
+- Are tickets in the correct columns?
+  - Can any issues be closed / moved to the closed column?
+  - Are all in progress tickets properly assigned and in the right column?
+  - Are things requiring PO review identified and communicated to PO?
+  - Ditto for design review?
+- Does the board reflect the current state?  If the
+
+### Ready Column
+- Must always be kept populated
+  - Depth of at least one ticket per developer
+- Each ticket here must be actionable for the developers in the work cycle
+  - any open questions for the PO or the designer aren't blocking the work (if so, ticket goes in backlog) ... and are indicated in the ticket
+  - needed background information available
+  - volume of text in ticket isn't so overwhelming that developer backs away slowly. (Adding structure so it's more skimmable can help)
+  - it's okay to add comments to the ticket like "talk to me before you start coding for this" or "this will require some familiarity with xxx"
+- Check board for new tickets created by others, ticket order in columns, etc.
+- make sure PRs are reviewed promptly, and well.
+
+### Pull Requests
+
+_Quick Review_: It is up to the TL to ensure that PRs are reviewed quickly.  If the TL does a fair number of reviews, this also helps ensure new code is aligned with workcycle coding goals (see below).
+- Ideally within 2-3 hours
+- More than 24 hours before a review is not good.
+
+_Identify when PO or Designer Should Review Work_: Sometimes the TL has a better feel than the executing developer for when work should get PO sign off.  If you realize this while reviewing the PR, the TL should indicate PO sign off is needed (possibly change the PR to "draft" or at least make it clear in the title that PO sign off is needed (prefix "[hold for PO approval]" or some such)) and be sure to communicate with the PO that their input is desired.
+
+_Before/After Screenshots in PR descriptions Save Time_: In the ETD work-cycle in 2020Q3, the PO was often able to sign off on PRs just from screen shots - faster for the PO to review, and faster turnaround on the PR.  It can also help your fellow developers "see" that your changes did the desired thing:  did change, or didn't change, what is showing in the UI.
+- Don't be shy about asking for these if they'll be helpful.
+
+# What Does a TL _AVOID_?
+Let's repeat these things here:
+- TL is a full-time job; I don't believe you can be a good TL and a developer at the same time because being TL takes at least 70% of work time, and the remaining time is usually broken up into a few minutes here and there.
```

Comment:
Duplicative; this point was already made earlier in the document.

Reply:
Looks like this section is more of an summary overview and so its ok if it is duplicative?  But maybe this point could be shortened in sumamry

Reply:
I'm OK with the intentional repetition here to help wrap the piece up.